README.md

```diff
@@ -106,6 +109,46 @@ We use the basic feature branch GIT flow. Create a feature branch off of master
 request. Make branch names and commits descriptive. A merge request should solve one problem,
 not many. 
 
+## Offline installation
+
+To install CALDERA on a server without internet access, `pip` can be used to download the CALDERA dependencies
+from a machine with internet access.  Once the dependencies are downloaded, they can be copied to the
+offline machine and installed.
+
+The internet machine's platform and python version should match offline server.  For example, if the 
+the offline target machine runs Python 3.6 on CentOS 7, then Python3.6 and CentOS 7 should be used to perform 
+the packaging to minimize problems.
+
+```bash
+git clone --recursive https://github.com/mitre/caldera.git
+mkdir caldera/python_deps
+pip download -r caldera/requirements.txt --dest caldera/python_deps
+```
+
+If you want to use the adversary plugin, be sure to download those dependencies as well:
```

Comment:
move this section to the adversary plugin readme, under an 'offline' section

Reply:
54732611b43cdeecf556daa1c4c6421dce5c7d73